src/rules/prefer-absolute-version-dependencies.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const areVersRangesValid = require('./../validators/dependency-audit').areVersRangesValid;
+const LintIssue = require('./../LintIssue');
+const lintId = 'prefer-absolute-version-dependencies';
+const nodeName = 'dependencies';
+const message = 'You are using an invalid version range. Please use absolute versions.';
+const ruleType = 'dependencies-version-range';
+
+const lint = function(packageJsonData, lintType) {
+  const rangeSpecifier = '=';
```

Comment:
What are your thoughts on also allowing absolute ranges that don't begin with an equals sign? Equals is assumed if no operator is specified.  https://docs.npmjs.com/misc/semver#versions

Note: the underlying helper function doesn't work well without a leading character.